src/utils/ui.js

```diff
@@ -2,6 +2,13 @@ import { getState, saveState } from './state';
 import { makeElement } from './misc';
 import * as player from './player';
 
+let windowNames = {
```

Comment:
1) Can we rename this to `WindowNames`? I'm a fan of differentiating constant variables, i.e. literal strings/ints/etc set in the code and never changed (unlike a lot of our instances of `const varName = ...`), naming from other variables. Some codebases also use the `WINDOW_NAMES` naming standard, but that's slightly more of a hassle to type, so I'm fine with `WindowNames`.

2) This can be `const` :)

Reply:
Sounds good, done!